Mathlib/LinearAlgebra/Matrix/GeneralLinearGroup/Defs.lean

```diff
@@ -69,7 +70,7 @@ def det : GL n R →* Rˣ where
   map_one' := Units.ext det_one
   map_mul' _ _ := Units.ext <| det_mul _ _
 
-/-- The `GL n R` and `Matrix.GeneralLinearGroup R n` groups are multiplicatively equivalent -/
+/-- The groups `GL n R` and `Matrix.GeneralLinearGroup R n` are multiplicatively equivalent -/
```

Comment:
It should be
```suggestion
/-- The groups `GL n R` and `LinearMap.GeneralLinearGroup R n` are multiplicatively equivalent -/
```
right? Maybe write "`GL n R` (notation for Matrix.GeneralLinearGroup R n`)" for clarification.